jujugui/static/gui/src/app/components/deployment-flow/test-deployment-flow.js

```diff
@@ -23,118 +23,113 @@ var juju = {components: {}}; // eslint-disable-line no-unused-vars
 chai.config.includeStack = true;
 chai.config.truncateThreshold = 0;
 
+/**
+  Convenience function for quickly and easily creating a DeploymentFlow
+  component. It accept a props param, which can be used to change defaults (on
+  required props) or provide values (on optional props).
+ */
+const createDeploymentFlow = (props = {}) => {
```

Comment:
This does not seem to me a big change in how we are testing the component. It seems that we are abstracting out the instantiation of the component instead, and this allows for avoiding a lot of repetition in the tests. It seems a good idea to me.